src/CSSBattle/11_EyeOfSauron/ShaoKiat.html

```diff
@@ -0,0 +1,74 @@
+<body>
+  <div class="circle orange middle" id="outer-circle"></div>
+  <div class="circle black middle" id="middle-circle"></div>
+  <div class="circle red middle" id="inner-circle"></div>
+  <div class="semi-circle orange left outer">
+    <div class="semi-circle black inner"></div>
+  </div>
+  <div class="semi-circle orange right outer" id="invert">
+    <div class="semi-circle black inner" id="invert"></div>
```

Comment:
id can be removed since it is not used; id should be unique

Reply:
69acf78